src/main/resources/application-test.yaml

```diff
@@ -25,9 +25,6 @@ spring:
     clean-disabled: false
   thymeleaf:
     prefix: classpath:/templates/
-  jpa:
-    hibernate:
-      ddl-auto: create-drop
```

Comment:
I thought this just dropped the tables at the end of testing.  Is there something that cleans up data from tests?

Reply:
That being said, I'm not super clear on how this config affects things.

Reply:
See https://docs.spring.io/spring-boot/docs/1.1.0.M1/reference/html/howto-database-initialization.html#howto-initialize-a-database-using-hibernate. This was the root cause of https://www.pivotaltracker.com/story/show/186402340.